typescript-forms/form-events.ts

```diff
@@ -0,0 +1,40 @@
+function handleFocus(event: Event): void {
+  console.log('Focus event fired');
+  const eventTarget = event.target as HTMLInputElement | HTMLTextAreaElement;
+  console.log(eventTarget.name);
+}
+
+function handleBlur(event: Event): void {
+  console.log('Blur event fired');
+  const eventTarget = event.target as HTMLInputElement | HTMLTextAreaElement;
+  console.log(eventTarget.name);
+}
+
+function handleInput(event: Event): void {
+  const eventTarget = event.target as HTMLInputElement | HTMLTextAreaElement;
+  console.log(eventTarget.name, eventTarget.value);
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const input1 = document.querySelector('input[name="name"]');
+  const input2 = document.querySelector('input[name="email"]');
+  const textArea = document.querySelector('textArea[name="message"]');
+
```

Comment:
You don't need the `DOMContentLoaded` event listener and I would avoid adding it at all. You can potentially run into issues where you script doesn't run if there are asynchronous scripts that should run beforehand which can cause errors in your code.